modules/Bio/EnsEMBL/Production/Pipeline/GPAD/LoadFile.pm

```diff
@@ -22,7 +22,7 @@ Bio::EnsEMBL::Production::Pipeline::GPAD::LoadFile;
 
 =head1 AUTHOR
 
-ckong@ebi.ac.uk
+ckong@ebi.ac.uk and maurel@ebi.ac.uk
```

Comment:
Is it not Ensembl policy to not have named authors? I think EG worked differently, and this might be a vestigial tail from those times.

Reply:
I don't think it is. I could remove this